### ISSUE TITLE
Refactor documentation structure

### ISSUE THREAD

**Issue by JacquesBonet** [strongloop/loopback-next#5104]
## Suggestion

In my opinion, the structuration of Loopback V4 documentation is very very bad.

I never find what I'm looking. When I use Loopback V3 documentation, I find the information very quickly.

I think a more traditional structuration like with Loopback V3 will be far better.

**Comment by achrinza** [comment]
HI @JacquesBonet, thanks for the suggestion! Could you clarify which parts of the LoopBack 3 docs would you like to see in the LoopBack 4 docs? Or maybe what is difficult to find in the lb4 docs?

**Comment by JacquesBonet** [comment]
May be its not the doc the problem but the Authentication/Authorization mecanism.

I understand its a very difficult task to explain how works Authentication and Authorization with all these interfaces, providers, services, ... to implements.

I like very much Loopback except these two modules which are over complicated to use.
Also they don't match with rest-crud.

May be a little more convention and code separation could be benefic.

I'll try to do my own implementation (surely to much simplified to be integrated to loopback).

**Comment by JacquesBonet** [comment]
An example of problem of bad documents structuration.

I want to have information about how to filter request.

I look to all the chapters of the documentation, I see nothing.
Now I use search engine, enter filter keyword, and the informations displayer on the firt pas are:

Cassandra Connector
Dependency Injection Decorator
Controller Generator
Open API Decorator
Relation
Migrating Custom Model
Parsing Request
Has many Relation
Has One relation Cloudant

Nothing correspond to what I want.

**Comment by dougal83** [comment]
I appreciate your frustration, personally I've used lb3 documentation where applicable to find my way around in the past.  Chances are the docs that you are looking for are in progress:
[Working with data](https://github.com/strongloop/loopback-next/blob/dd99448cc2b943c70a459ce6c31ca0c310d768db/docs/site/Working-with-data.md)

The above is going live soon, might have what you are after?

**Comment by achrinza** [comment]
Thanks for the suggestions, @JacquesBonet; I understand the frustration of not being able to find what you're looking for. In this case, we've just recently added the docs for Working with data.

I believe this can be part of our overall docs improvement (see #5113). Let's continue the discussion there so that we can consolidate areas that we need to touch on.

I've also quoted a key point from your remarks to help shape the direction of the refactor.
